mlflow/tracking/fluent.py

```diff
@@ -311,6 +456,21 @@ def log_artifact(local_path, artifact_path=None):
 
     :param local_path: Path to the file to write.
     :param artifact_path: If provided, the directory in ``artifact_uri`` to write to.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create a features.txt artifact file
+        features = "rooms, zipcode, median_price, school_rating, transport"
+        with open("features.txt", 'w') as f:
+            f.write(features)
+
+        # Use context manager to create a new run and write this file
+        # in a directory "features" under the root artifact_uri/features
+        with mlflow.start_run(run_name="My Runs"):
+            mlflow.log_artifact("features.txt", artifact_path="features")
```

Comment:
Should we add an example for `artifact_path=None`? Some users might misunderstand that `artifact_path` is a required argument.

Reply:
Good point. I have changed the log_artifact() to use that, while log_artifacts() shows to store under artifact_path=< path_name>